src/v1/sources/shopify/webhookTransformations/serverSideUtlis.js

```diff
@@ -0,0 +1,34 @@
+const { constructPayload } = require('../../../../v0/util');
+
+const {
+  lineItemsMappingJSON,
+  productMappingJSON,
+} = require('../../../../v0/sources/shopify/config');
+
+const getProductsFromLineItems = (lineItems, mapping) => {
+  if (!lineItems || lineItems.length === 0) {
+    return [];
+  }
+  const products = [];
+  lineItems.forEach((lineItem) => {
+    // const product = constructPayload(lineItem, lineItemsMappingJSON);
+    const product = constructPayload(lineItem, mapping);
+    products.push(product);
+  });
+  return products;
+};
+
+const createPropertiesForV2EcomEvent = (message) => {
```

Comment:
what's v2 ecom event ?

Reply:
Updated the name to `createPropertiesForEcomEventFromWebhook`, it is the e-commerce event received from the pixel based app.